pt-BR/action_controller_overview.md

```diff
@@ -19,16 +19,16 @@ After reading this guide, you will know:
 
 --------------------------------------------------------------------------------
 
-What Does a Controller Do?
+O que um `Controller` faz?
 --------------------------
 
-Action Controller is the C in [MVC](https://en.wikipedia.org/wiki/Model%E2%80%93view%E2%80%93controller). After the router has determined which controller to use for a request, the controller is responsible for making sense of the request, and producing the appropriate output. Luckily, Action Controller does most of the groundwork for you and uses smart conventions to make this as straightforward as possible.
+`ActionController` é o _C_ em [MVC](https://en.wikipedia.org/wiki/Model%E2%80%93view%E2%80%93controller). Após o `router` determinar qual _controller_ usar para a requisição, o _controller_ será responsavel por entender a requisição e retornar a reposta apropriada. Por sorte, `ActionController` faz a maior parte do trabalho fundamental pra você e usa convenções inteligentes pra fazer esse processo ser tão intuitivo quanto possível.
```

Comment:
Mesma sugestão acima, como não é código acredito que podemos tornar itálico.

O que acha? Também achei um pequeno typo.

```suggestion
*ActionController* é o _C_ em [MVC](https://en.wikipedia.org/wiki/Model%E2%80%93view%E2%80%93controller). Após o `router` determinar qual _controller_ usar para a requisição, o _controller_ será responsável por entender a requisição e retornar a resposta apropriada. Por sorte, *ActionController* faz a maior parte do trabalho pra você e usa convenções inteligentes pra fazer esse processo ser tão intuitivo quanto possível.
```